<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FanTestTool</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>822</width>
     <height>751</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QCheckBox" name="startup">
        <property name="text">
         <string>运行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoModeEnableBox">
        <property name="text">
         <string>自动</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>端口:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="serialPortsSelector">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>100</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>80</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="title">
         <string>手动：</string>
        </property>
        <widget class="QCheckBox" name="directionBox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>100</width>
           <height>50</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>正转</string>
         </property>
        </widget>
        <widget class="QComboBox" name="speedLevelCombox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>120</width>
           <height>30</height>
          </rect>
         </property>
         <property name="maxVisibleItems">
          <number>6</number>
         </property>
         <property name="maxCount">
          <number>6</number>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>200</horstretch>
          <verstretch>200</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>80</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="title">
         <string>自动</string>
        </property>
        <widget class="QLineEdit" name="configFilePathText">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>200</width>
           <height>30</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="configSettingBtn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>150</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>配置文件：</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="RecordSelectorBtn">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>输出文件：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="RecordFilePathText">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="loggerText">
      <property name="maximumSize">
       <size>
        <width>679</width>
        <height>500</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
